rudder-webapp/SOURCES/Makefile

```diff
@@ -57,11 +57,15 @@ localdepends: ./rudder-sources ./maven/bin/mvn ./rudder-users.xml ./rudder-doc .
 ./rudder-users.xml: ./rudder-sources
 	cp ./rudder-sources/rudder/rudder-web/src/main/resources/demo-rudder-users.xml ./rudder-users.xml
 
+./.gitignore: ./configuration-repository-gitignore
+	cp ./configuration-repository-gitignore ./.gitignore
```

Comment:
You can still call it configuration-repository-gitignore since there is a rename during the packaging phase (like for rudder-agent.init)